eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -99,12 +99,12 @@ stages:
       - template: ../jobs/vmr-build.yml
         parameters:
           # Changing the build name requires updating the referenced name in the source-build-sdk-diff-tests.yml pipeline
-          buildName: CentOSStream8_Online_CurrentSourceBuiltSdk
+          buildName: centosStream9_Online_CurrentSourceBuiltSdk
           isBuiltFromVmr: ${{ parameters.isBuiltFromVmr }}
           vmrBranch: ${{ variables.VmrBranch }}
           architecture: x64
           pool: ${{ parameters.pool_Linux }}
-          container: ${{ variables.centOSStream8Container }}
+          container: ${{ variables.centosStream9Container }}
```

Comment:
This is going to cause the same issue mentioned in https://github.com/dotnet/installer/pull/19120#pullrequestreview-1976742435.

Reply:
We will need to upload the Stream 9 SDK/artifacts before merging.